contracts/ExitFormat.sol

```diff
@@ -23,14 +18,35 @@ library ExitFormat {
 
     // A SingleAssetExit specifies
     // * an asset address (0 implies the native asset of the chain: on mainnet, this is ETH)
-    // * custom metadata (optional field, can be zero bytes). This might specify how to transfer this particular asset (e.g. target an "ERC20.transfer"' method)
+    // * custom tokenMetadata
+    //   containing
+    //   - AssetType enum value
+    //   - metadata for that token
     // * an allocations array
     struct SingleAssetExit {
         address asset;
-        bytes metadata;
+        TokenMetadata tokenMetadata;
```

Comment:
I think we discussed changing this to AssetMetadata?

Reply:
Changed